schunk_gripper_library/schunk_gripper_library/driver.py

```diff
@@ -404,6 +408,24 @@ async def check() -> bool:
                 return False
             return await check()
 
+    def show_gripper_specification(
+        self, scheduler: Scheduler | None = None
+    ) -> dict[str, float | str] | bool:
```

Comment:
Change that to always return a dictionary. Return an empty dict if the call fails.